src/site/content/en/progressive-web-apps/what-are-pwas/index.md

```diff
@@ -0,0 +1,135 @@
+  - progressive-web-apps
+draft: true
+---
+
+The web is an incredible platform. Its mix of ubiquity across devices and
+operating systems, its user-centered security model, and the fact that neither
+its specification nor its implementation is controlled by a single company makes
+the web a unique platform to develop software on. Combined with its inherent
+linkability, it's possible to search across it and share what you've found with
+anyone, anywhere. Whenever you go to a website, it's up-to-date, and your
+experience with that site can be as ephemeral or as permanent as you'd like. Web
+applications can reach _anyone, anywhere, on any device_ with a single codebase.
+
+Native applications, are known for being incredibly rich and reliable. They're
+ever-present, on home screens and docks and taskbars. They work regardless of
+network connection. They launch in their own standalone experience. They can
+read and write files from the local file system, access hardware connected via
+USB, serial or bluetooth, and even interact with data stored on your device,
+like contacts and calendar events. In native applications, you can do things
+like take pictures, see playing songs listed on the home screen, or control song
+playback while in another app. Native applications feel like _part_ of the
+device they run on.
+
+<figure class="w-figure w-figure--center">
+  <img src="./capabilities-reach.svg" style="max-width: 100%;"
+    alt="A graph illustrating the relative capabilites and reach of native apps, with high capabilities, web apps, with high reach, and progressive web apps, which have both high capabilities and high reach."/>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Capabilities vs reach of native apps, web app, and progressive web apps.
+  </figcaption>
+</figure>
+
+If you think about native apps and web apps in terms of capabilities and reach,
+native apps represent the best of capabilities whereas web apps represent the
+best of reach. So where do Progressive Web Apps fit in?
+
+Progressive Web Apps (PWA) are built and enhanced with modern APIs to deliver
+native-like capabilities, reliability, and installability while reaching
+_anyone, anywhere, on any device_ with a single codebase.
+
+## The three app pillars
+
+Progressive Web Apps are web applications that have been designed so they are
+capable, reliable, and installable. These three pillars transform them into an
+experience that feels like a native application.
+
+### Capable
+
+The web is quite capable in its own right today. You can build a hyper-local
+video chat app using WebRTC, geolocation, and push notifications. You can make
+that app installable and take those conversations virtual with WebGL and WebVR.
+With the introduction of Web Assembly, developers can tap into other ecosystems,
+like C, C++, and Rust, and bring decades of work and capabilities to the web
+too! [Squoosh.app](https://squoosh.app/), for instance, leveraged this for its
```

Comment:
Change exclamation mark to a period.
Change 'leveraged' to 'leverages'.

Reply:
```suggestion
too. [Squoosh.app](https://squoosh.app/), for instance, leverages this for its
```